### PR TITLE
Update AWS SDK to v2.2.6

### PR DESCRIPTION
**Pull request by nyaaao**

### CODE REVIEW

File: package.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
 Package.describe({
   summary: "SDK for AWS services including Amazon S3, Amazon EC2, DynamoDB, and Amazon SWF",
-  version: '2.1.47_1',
+  version: '2.2.6',
```
Comment (mitar) [inline package.js]: Please use `_1` suffix.

**Comment by mitar** [comment]
Please make a comment to notify me once you fix it.

**Comment by nyaaao** [comment]
fixed

**Comment by mitar** [comment]
Great. Thanks!

**Pull request merged**